### ISSUE TITLE
Wrong font size in hotkeys_popup

### ISSUE THREAD

**Issue by DenisG20** [awesomeWM/awesome#3158]
Output of `awesome --version`:

awesome v4.3 (Too long)
 • Compiled against Lua 5.3.5 (running with Lua 5.3)
 • D-Bus support: ✔
 • execinfo support: ✔
 • xcb-randr version: 1.6
 • LGI version: 0.9.2

**How to reproduce the issue:**

Add to your theme the following:
theme.hotkeys_font                              = "Monospace 12"
theme.hotkeys_description_font         = "Monospace 14"

**Actual result:**
Columns are truncated

![image](https://user-images.githubusercontent.com/70662493/92026868-ab375f00-ed2f-11ea-8af9-d975c388f232.png)

**Expected result:**

Columns height calculated correctly and pagination if necessary (a larger popup would be nice) as with:

![image](https://user-images.githubusercontent.com/70662493/92027191-39134a00-ed30-11ea-81ac-5ade6d7f234f.png)

I normally rule out tmux keys but for this report I uses rc.lua and the default theme as distributed with no other modification. I also tried to set hotkeys_font and hotkeys_description_font via beautiful.get() with exactly (as expected) the same result. My eyes cannot read the original font size on a 4k monitor it is simply too small...

Otherwise, after playing with Awesome for a couple of days, I have to agree it is truly AWESOME...

Denis

**Comment by slickroot** [comment]
Somehow the screenshots are really small.

have you tried `monospace` instead of `Monospace`? I don't know if it makes a difference.

have you tried setting `Xft.dpi: 255` on your `.Xresources`?

**Comment by DenisG20** [comment]
Apologies for the small images
Here is the view with the hotkeys_font etc set as I wish
![image](https://user-images.githubusercontent.com/70662493/92054192-fb2c1b00-ed5b-11ea-8dc9-5619002a35c8.png)
And here the shot without the theme 2 assignments
![image](https://user-images.githubusercontent.com/70662493/92055033-3890a880-ed5c-11ea-94de-b32832c124f0.png)
You can see clearly that the columns are simply truncated.

I have no dpi issue as I am sure you can see in the open terminal behind the popup.
monospace or Monospace makes no difference.

**Comment by actionless** [comment]
@DenisG20 please test with this patch: https://github.com/awesomeWM/awesome/pull/3160/files

**Comment by DenisG20** [comment]
@actionless Sorry for all the bother. I saw that your patch did not match the installed code... I went on to install the github master (which I realised I should have done before reporting this problem) and with the master as is all is well.

I am afraid I wasted your time. Everything works as expected without your patch...

Here is my current version:

awesome v4.3-862-g7a759432d-dirty (Too long)
 • Compiled against Lua 5.3.5 (running with Lua 5.3)
 • API level: 4
 • D-Bus support: yes
 • xcb-errors support: no
 • execinfo support: yes
 • xcb-randr version: 1.6
 • LGI version: 0.9.2
 • Transparency enabled: yes
 • Custom search paths: no
